src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -557,6 +557,17 @@ public static Path getPath(String uriString) {
         }
     }
 
+    /**
+     * Gets the absolute Path name with the URI marker, handling the special case of the default file system by removing
+     * the file:// prefix.
+     *
+     * @param path path to get the absolute name.
+     * @return a String with the absolute name.
```

Comment:
"a String with the absolute name, and the file:// protocol removed, if it was present"

Reply:
Done.